data/src/main/java/com/microsoft/azure/kusto/data/exceptions/DataWebException.java

```diff
@@ -9,7 +9,7 @@
 public class DataWebException extends Exception{
 
     private String message;
-    private HttpResponse httpResponse;
+    private final HttpResponse httpResponse;
```

Comment:
This field is never used, so I suggest we remove it altogether.

Reply:
again its for the usage of customers